examples/Querying Chromeleon Data.ipynb

```diff
@@ -0,0 +1,276 @@
+    "                  {\"wildcard\": {\"file.path\": \"*SEQUENCE_NAME.seq*\"}}\n",
+    "                  ]\n",
+    "          }\n",
+    "      },\n",
+    "    \"_source\": [ \"fileId\", \"filePath\", \"labels\"],\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "475f7c0a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "payload"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b50996d",
+   "metadata": {},
+   "source": [
+    "## Run Search Request and Display Results\n",
+    "\n",
+    "We will now use the requests library to make a request to the TDP API.  We have previously configured the connection variables as well as the query we are executing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d4e348c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "request = requests.post(search_url, json=payload, headers=headers)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cf4cdb54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = request.json()['hits']['hits']\n",
+    "\n",
+    "# print the first result to see structure\n",
+    "result[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff743f09",
+   "metadata": {},
+   "source": [
+    "We can look at all the run names of the sequence by printing the run labels out:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "82a62802",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for run in result:\n",
+    "  labels = run[\"_source\"][\"labels\"]\n",
+    "  run_name = next(item[\"value\"] for item in labels if item[\"name\"] == \"run_name\")\n",
+    "  print(run_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "45da4263",
+   "metadata": {},
+   "source": [
+    "If you want to filter on specific runs, you could write code here to do that."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec78b2e1",
+   "metadata": {},
+   "source": [
+    "## Retrieve the Amounts of Chemicals in one Sequence"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e5ba086d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "first_file_id = result[0][\"_source\"][\"fileId\"]\n",
+    "print(first_file_id)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f2810413",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "first_file = requests.get(retrieve_file+\"?fileId=\"+first_file_id, headers=headers)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "102f1bbb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IDS_info = json.loads(first_file.text)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "696189e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
I might have just chosen the wrong data for my test, but I don't think this worked as expected for me. Wondering if there's a way to narrow down the query to ensure only compatible results are returned, but I didn't look closely.

![image](https://github.com/tetrascience/ts-scientific-data-cloud-api-examples/assets/97319818/37fdabcf-d7be-4576-801d-e97b6dc6e0de)